docs/built-in-functions.md

```diff
@@ -170,7 +170,9 @@ There are two groups of built-in functions:
         unit: ms
 ```
 
-- LongTrimUnit/DoubleTrimUnit - Trims the specified unit string from the end of value of the metricName and returns Long/Double respectively. E.g. LongTrimUnit(Value,ms) - trims `ms` from  the end of value of metric name `Value`
+- LongTrimUnit/DoubleTrimUnit - Trims the specified units string from the end of value of the metricName and returns Long/Double respectively. 
+E.g. LongTrimUnit(Value,ms) - trims `ms` from  the end of value of metric name `Value`.
+E.g. LongTrimUnit(Value,s,ms) - trims `ms` or `s` (whichever is longest match) from  the end of value of metric name `Value`.
```

Comment:
Hm, I think we'll need smarter logic here.

The thing being monitored may randomly use ms, s, sec, ns, min... for the same metric. From the agent it would be expected that it always produces values for some metric using the same unit, say `ms` (that unit could be one of parameters). 

So, I think we need special LongTrimTimeUnit function that knows about possible time units and knows how to convert any of them into time unit passed as param.